LottieSample/src/main/kotlin/com/airbnb/lottie/samples/LottieFontViewGroup.kt

```diff
@@ -151,17 +149,17 @@ class LottieFontViewGroup @JvmOverloads constructor(
         //         break;
         // }
         val fileName = "Mobilo/$letter.json"
-        if (compositionMap.containsKey(fileName)) {
```

Comment:
1、time complexity of containsKey function is same as get function, so we can use the result of get function to assert whether it meet the condition.
2、be careful to use !! operator, it may cause crash, although here don‘t occur crash, but it looks ugly.